src/main/conf/ds-cumulus-export-default-mapping.yml

```diff
@@ -96,10 +99,48 @@ digisam:
           replacement: "https://kb-images.kb.dk/DAMJP2/$1/"
           lineBreakIsMulti: false
 
-        - source: "Horizontal Pixels"
+        - source: "Horizontal Pixels"  # COP: part of Dimension
           dest: "width_pixels"
           destType: "long"
 
-        - source: "Vertical Pixels"
+        - source: "Vertical Pixels"  # COP: part of Dimension
           dest: "height_pixels"
           destType: "long"
+
+        - source: "Record Name"  # COP: Id
+          dest: "record_name"
+
+        - source: "Materialebetegnelse"  # COP: Ressourcetype 1 (If missing: Generel materialebetegnelse below is shown)
+          dest: "material_1"
+
+        - source: "Generel materialebetegnelse"  # COP: Ressourcetype 2
+          dest: "material_2"
+
+        - source: "Opstilling"  # COP: Opstilling
+          dest: "archive" # Better tag needed
+
+        - source: "Lokalitet"  # COP: Lokalitet
+          dest: "location"
+
+        - source: "Accessionsnr"  # COP: Accessionsnummer
+          dest: "accession_number"
+
+        - source: "Keywords"  # COP: Emne
+          dest: "topic"  # or replace keyword?
+
+        - source: "Note"  # COP: Kommentar
+          dest: "note"
+
+        - source: "Person" # COP: Person (Known to change format, related to "Ophav")
+          dest: "person"
+
+        - source: "URL"  # COP: Relateret (The link to main page for pictures)
+          dest: "related_url"
+          destType: "url"
+          pattern: "http://www.kb.dk/(.*)"
+          replacement: "http://www5.kb.dk/$1"  # is this COP only??
+          verifyURL: true
+
+        - source: "URL-tekst" # COP: Releateret (The text to display for the link above)
+          dest: "related_url_text"
```

Comment:
This presents a bit of a problem as we can have multiple `related_url` and because Solr does not directly support value-pairs inside of fields. We could do it markdown-style by storing `[URL-tekst](URL)`?